source/manage/server-admin-gui.rst

```diff
@@ -0,0 +1,72 @@
+--------------------------------------------------------------------------------
+PMM Settings Page
+--------------------------------------------------------------------------------
+
+|pmm-settings| is a special page dedicated to configuring a number of PMM
+options. It can be accessed through the main menu:
+
+   .. figure:: ../.res/graphics/png/pmm-add-instance.png
+
+      Choosing the |pmm| *Settings* menu entry
+
+|pmm-settings| page consists of the following sections, which allow to configure
+different aspects of the PMM Server:
+
+.. contents::
+   :local:
+   :depth: 1
+
+Settings
+================================================================================
+
+*Settings* section allows you to change `metrics resolution <https://www.percona.com/doc/percona-monitoring-and-management/2.x/faq.html#what-resolution-is-used-for-metrics>`_, `data retention <https://www.percona.com/doc/percona-monitoring-and-management/2.x/faq.html#how-to-control-data-retention-for-pmm>`_,
+as well as configure `telemetry <https://www.percona.com/doc/percona-monitoring-and-management/2.x/glossary-terminology.html#Telemetry>`_ and automatic checking for `updates <https://www.percona.com/doc/percona-monitoring-and-management/2.x/glossary-terminology.html#PMM-Version>`_:
```

Comment:
I haven't seen a PR to add telemetry to the glossary...right now this links to nothing.